EventFilter/CastorRawToDigi/plugins/CastorRawToDigi.cc

```diff
@@ -21,7 +21,7 @@ using namespace std;
 CastorRawToDigi::CastorRawToDigi(edm::ParameterSet const& conf):
   dataTag_(conf.getParameter<edm::InputTag>("InputLabel")),
   unpacker_(conf.getParameter<int>("CastorFirstFED"),conf.getParameter<int>("firstSample"),conf.getParameter<int>("lastSample")),
-  zdcunpacker_(conf.getParameter<int>("ZDCFirstFED"),conf.getParameter<int>("firstSample"),conf.getParameter<int>("lastSample")),
+  zdcunpacker_(conf.getParameter<int>("CastorFirstFED"),conf.getParameter<int>("firstSample"),conf.getParameter<int>("lastSample")),
```

Comment:
It is used as sourceIdOffset_:
https://github.com/BetterWang/cmssw/blob/ZDCunpacker_update/EventFilter/CastorRawToDigi/src/ZdcUnpacker.cc#L84
This is because the FED index is with respect to the first FED number.
In this case, the actual ZDC FED is 693. When referred in the ElectronicId, the uccid=693-690=3, where 690 is the FED number of the first Castor FED.